docs/knowledge_base/lockfile-generation.md

```diff
@@ -12,7 +12,7 @@ The Phylum CLI can generate a lockfile when it is given a manifest file.
 | ------------- | ---------        | -------------               |
 | `npm`         | `package.json`   | [`npm`][npm]                |
 | `yarn`        | `package.json`   | [`yarn`][yarn]              |
-| `pip`         | `requirements*.txt` <br/> `requirements.in` <br/> `setup.py` <br/> `pyproject.toml` | `pip-compile` (from [`pip-tools`][pip-tools]) |
+| `pip`         | `requirements*.txt` <br/> `requirements.in` <br/> `setup.py` <br/> `pyproject.toml` | [`pip3`][pip3] version 23.0.0+ |
```

Comment:
There is a good reason why we use `pip3`. We tested it for the extensions and `pip3` was found to be much more available than `pip` because some distributions still shipped `pip` as python 2 back when the pip extension was written.

I don't think there's any point in documenting this as `pip` when the code itself calls `pip3`.

Reply:
The assertion is that it is more confusing to users to read `pip3` than `pip`. The link here points to the `pip` documentation, which does not make _any_ mention of `pip3`. The other changes suggested in this PR are to switch from calling `pip3` to calling `python -m pip` instead.